opnode/rollup/driver/state.go

```diff
@@ -137,6 +141,7 @@ func (s *state) loop() {
 		// case <-l1Poll.C:
 		// case <-l2Poll.C:
 		case <-s.done:
+			atomic.AddUint32(&s.closed, 1)
```

Comment:
agreed, temporary work around to make tests stable: when pausing go routines during debugging I hit a crash because the logger would run after closing